ml.net/InclusiveCodeReviews.ConsoleApp/ModelBuilder.cs

```diff
@@ -55,6 +55,10 @@ public static IEstimator<ITransformer> BuildTrainingPipeline(MLContext mlContext
             var dataProcessPipeline = mlContext.Transforms.Conversion.MapValueToKey("isnegative", "isnegative")
                                       .Append(mlContext.Transforms.Text.FeaturizeText("text_tf", new TextFeaturizingEstimator.Options
                                       {
+                                          CaseMode = TextNormalizingEstimator.CaseMode.Lower,
+                                          //NOTE: not exportable to ONNX
+                                          //KeepNumbers = false,
+                                          //KeepPunctuations = false,
                                           KeepDiacritics = true,
                                       }, "text"))
```

Comment:
@luisquintanilla, I found these commented options are maybe not exportable to `.onnx` yet?

They both cause an exception such as:
```
Unhandled exception. System.Collections.Generic.KeyNotFoundException: The given key 'text_TextNormalizer' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Microsoft.ML.Model.OnnxConverter.OnnxContextImpl.GetVariableName(String colName)
   at Microsoft.ML.Transforms.Text.WordTokenizingTransformer.Mapper.SaveAsOnnx(OnnxContext ctx)
   at Microsoft.ML.Data.RowToRowMapperTransform.Microsoft.ML.Model.OnnxConverter.ISaveAsOnnx.SaveAsOnnx(OnnxContext ctx)
   at Microsoft.ML.Model.OnnxConverter.SaveOnnxCommand.ConvertTransformListToOnnxModel(OnnxContextImpl ctx, IChannel ch, IDataView inputData, IDataView outputData, LinkedList`1 transforms, HashSet`1 inputColumnNamesToDrop, HashSet`1 outputColumnNamesToDrop)
   at Microsoft.ML.OnnxExportExtensions.ConvertToOnnxProtobufCore(IHostEnvironment env, OnnxContextImpl ctx, ITransformer transform, IDataView inputData, String[] outputColumnNamesToKeep)
   at Microsoft.ML.OnnxExportExtensions.ConvertToOnnxProtobuf(ModelOperationsCatalog catalog, ITransformer transfor   at Microsoft.ML.OnnxExportExtensions.ConvertToOnnx(ModelOperationsCatalog catalog, ITransformer transform, IDataView inputData, Stream stream)
   at InclusiveCodeReviews.ConsoleApp.ModelBuilder.SaveModel(MLContext mlContext, IDataView dataView, ITransformer 
mlModel, String modelRelativePath, DataViewSchema modelInputSchema) in C:\src\inclusive-code-reviews-ml\ml.net\InclusiveCodeReviews.ConsoleApp\ModelBuilder.cs:line 102
   at InclusiveCodeReviews.ConsoleApp.ModelBuilder.CreateModel() in C:\src\inclusive-code-reviews-ml\ml.net\InclusiveCodeReviews.ConsoleApp\ModelBuilder.cs:line 49
   at Program.<Main>$(String[] args) in C:\src\inclusive-code-reviews-ml\ml.net\InclusiveCodeReviews.Convert\Program.cs:line 4
```